Brisk/Controllers/FileDuplicateViewController.swift

```diff
@@ -0,0 +1,80 @@
+import Alamofire
+import AppKit
+import Sonar
+
+final class FileDuplicateViewController: ViewController {
+    @IBOutlet private var progressIndicator: NSProgressIndicator!
+    @IBOutlet private var radarIDTextField: NSTextField!
+    @IBOutlet private var searchButton: NSButton!
+
+    @IBAction private func searchForOpenRadar(_ sender: NSButton) {
+        let setLoading: (Bool) -> Void = { [weak self] loading in
+            self?.progressIndicator.isLoading = loading
+            self?.radarIDTextField.isEnabled = !loading
+            self?.searchButton.isEnabled = !loading
+        }
+
+        setLoading(true)
+        let id = radarID(from: self.radarIDTextField.stringValue)!
+        let url = URL(string: "https://openradar.appspot.com/api/radar?number=\(id)")!
+        Alamofire.request(url)
+            .validate()
+            .responseJSON { [weak self] result in
+                setLoading(false)
+
+                guard let json = result.value as? [String: Any],
+                    let result = json["result"] as? [String: Any], !result.isEmpty else
+                {
+                    self?.showError(title: "No OpenRadar found",
+                                    message: "Couldn't find an OpenRadar with ID #\(id)")
+                    return
+                }
```

Comment:
I got this error when little snitch blocked the request, think it's worth separating that case?